source/_integrations/melcloud.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: MELCloud
+description: MELCloud integration
+logo: home-assistant.png
+ha_category:
+  - Climate
+ha_release: 0.106
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@vilppuvuorinen'
+---
+
+The `melcloud` integration integrates Mitsubishi Electric's MELCloud enabled devices into Home Assistant.
```

Comment:
Is there some site that could be linked here?

Reply:
Linked.